cfme/intelligence/reports/ui_elements.py

```diff
@@ -608,6 +612,7 @@ def combo(self):
 
 @fill.method((DashboardWidgetSelector, Sequence))
 def _fill_dws_seq(dws, seq):
+    # Doesn't work now because of bz 1287767
```

Comment:
This will work in the future when the bug is resolved? As this code isn't used much, this is probably OK, but, we do have the option of putting a bug test in there and skipping if the bug is not resolved, or raising an exception.